files/besu/README.md

```diff
@@ -129,19 +129,24 @@ There is an additional erc20 token example that you can also test with: executin
 This can be verified from the `data` field of the `logs` which is `1`.
 
 ### iii. Smart Contracts & DApps <a name="poa-network-dapps"></a>
-- Once you have a network up and running from above, install [metamask](https://metamask.io/) as an extension in your browser
-- Once you have setup your own private account, select 'My Accounts' by clicking on the avatar pic and then 'Import Account' and enter the valid private_key
-- You can now deploy contracts and connect DApps to the network. 
-![Image dapp](./static/qs-dapp.png)
-
-As seen in the architecture overview diagram you can extend the network with monitoring, logging, smart contracts, DApps and so on
-
 As an example we've included the Truffle Pet-Shop Dapp in the `dapps` folder and here is a [video tutorial](https://www.youtube.com/watch?v=_3E9FRJldj8) you 
 can follow of deployment to the network and using it. Please import the private key `0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3` to
 Metmask **before** proceeding to build and run the DApp with `run-dapp.sh`. Behind the scenes, this has used a smart contract that is compiled and then 
 deployed (via a migration) to our test network. The source code for the smart contract and the DApp can be found in the folder `dapps/pet-shop`
 
+![Image dapp](./static/qs-dapp.png)
+
+As seen in the architecture overview diagram you can extend the network with monitoring, logging, smart contracts, DApps and so on
+
+- Once you have a network up and running from above, install [metamask](https://metamask.io/) as an extension in your browser
+- Once you have setup your own private account, select 'My Accounts' by clicking on the avatar pic and then 'Import Account' and enter the private key `0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3`
```

Comment:
It may be worth adding big warnings about the use of this test account as we do in the Besu doc because we noticed some real ether transfered to them on mainnet, meaning people do sometimes misunderstand that it should only be used on test network.

Reply:
Done, have added a warning section there in bold now with the same contents as besu-docs

Reply:
Ready to re review @NicolasMassart